test/object-deep-map-values/index.js

```diff
@@ -0,0 +1,27 @@
+const test = require('../util/test')(__filename);
+const deepMapValues = require('../../packages/object-deep-map-values');
+
+const squareFn = (value) => value * value;
+const concatFn = (value, key) => key + value;
+
+test('correctly maps using value argument', (t) => {
```

Comment:
Please add tests for a) a flat object with no nesting b) failing cases (e.g missing arguments or arguments with wrong types).

Reply:
a: This was already being tested
b: Added